src/gc/os.d

```diff
@@ -49,6 +49,48 @@ else version (Posix)
     version (CRuntime_UClibc) import core.sys.linux.sys.mman : MAP_ANON;
     import core.stdc.stdlib;
 
+
+    /**
+     * Possible results for the wait_pid() function.
+     */
+    enum WRes
+    {
+     DONE, /// The process has finished successfully
+     RUNNING, /// The process is still running
+     ERROR /// There was an error waiting for the process
```

Comment:
* Indentation is wrong
* The standard D style is to use lower case for enum members.